brave/brave-ios
# Display ads notifications UI from rewards

Fixes https://github.com/brave/brave-rewards-ios/issues/196

## Submitter Checklist:


## Test Plan:

- Enable rewards by removing NO_REWARDS build flag
- Create a wallet and verify ads is enabled
- Browse a couple websites
- Wait until an ad appears
- Clicking the ad should open the ad URL

## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
